operator/pkg/charts/charts.go

```diff
@@ -282,6 +282,12 @@ func updateInfraChartsFromInstall(in *v1beta1.Installation, clusterConfig *k0sv1
 					return nil, fmt.Errorf("set helm values velero.configuration: %w", err)
 				}
 
+				podVolumePath := filepath.Join(provider.EmbeddedClusterK0sSubDir(), "kubelet/pods")
+				newVals, err = helm.SetValue(newVals, "nodeAgent.podVolumePath", podVolumePath)
```

Comment:
this is not directly related but is a bug i found where this is being reverted on upgrade